app/components/Nav/Main/MainNavigator.js

```diff
@@ -700,9 +706,9 @@ const MainNavigator = () => (
     />
     <Stack.Screen name="AddBookmarkView" component={AddBookmarkView} />
     <Stack.Screen name="OfflineModeView" component={OfflineModeView} />
+    <Stack.Screen name="NotificationsView" component={NotificationsModeView} />
```

Comment:
Can we use the constant instead?
```suggestion
    <Stack.Screen name={Routes.NOTIFICATIONS.VIEW} component={NotificationsModeView} />
```